estate/views/estate_menus.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estate">
+        <menuitem id="estate_first_level_menu" name="Advertisements">
+            <menuitem id="estate_property_model_menu_action" action="estate_property_action" />
+        </menuitem>
+        <menuitem id="estate_settings" name="Settings">
+            <menuitem id="estate_property_type_model_menu_action"
+                action="estate_property_type_action" />
+            <menuitem id="estate_property_tag_model_menu_action"
+                action="estate_property_tag_action" />
+        </menuitem>
+    </menuitem>
```

Comment:
for the exercise: make one line per menuitems, not using encapsulation
but using parent_id instead.
this will help you when adding menuitems on bridge modules or modules depending on the module where you want to put a new menu item